packages/tools/fluid-runner/README.md

```diff
@@ -31,10 +31,35 @@ fluidRunner({ /* IFluidFileConverter implementation here */ });
 The input file is expected to be an ODSP snapshot.
 For some examples, see the files in the [localOdspSnapshots folder](./src/test/localOdspSnapshots).
 
+### Telemetry format
+There is an optional command line option `telemetryFormat` that allows you to specify the telemetry output format. The naming provided to this option is strict and must match an option in [OutputFormat](./src/logger/fileLogger.ts).
+The default format is currently `JSON`
+
+### Additional telemetry properties
+There is an optional command line option `telemetryProp` that allows you to specify additional properties that will be added to every telemetry entry. The format follows these rules:
+- every key must be a string
+- values may be either a string or a number
+- a value may not be empty
+
+Example of valid usages:
+```
+--telemetryProp prop1 value1 --telemetryProp prop2 10.5
+--telemetryProp "  prop1 " "   value1 " prop2 value2
```

Comment:
I'm not sure if this means - trailing spaces are removed, or a part of the content?
It feels like ideal solution - we do not do any transformations here (i.e. trailing spaces are left) and we do not use this as an example.
What would be nice to demostrate is usage of any special characters in the middle of the string, including "=" and "\"". That said, latter is likely terminal specific (for example "aaa\"bbb" will print \\ in Window's cmd)